src/commands/profiles/create.js

```diff
@@ -218,6 +218,17 @@ class ProfilesCreate extends BaseCommand {
     }
   }
 
+  async removeKeytarKeysByProfileId(profileId) {
```

Comment:
nit: Move this function definition to below where it's first called. Allows for reading the code from top-to-bottom more easily.

Reply:
Currently `removeKeytarKeysByProfileId` definition is right above where it's first called, Please let me know if I am missing something here.

Reply:
@ravali-rimmalapudi  - I guess the ask is to move **below** the first call?

Reply:
Thanks for the clarification. Updated accordingly.